src/java/picard/analysis/CollectAlignmentSummaryMetrics.java

```diff
@@ -69,7 +69,14 @@
         programGroup = Metrics.class
 )
 public class CollectAlignmentSummaryMetrics extends SinglePassSamProgram {
-    static final String USAGE = "Produces from a SAM or BAM a file containing summary alignment metrics";
+    static final String USAGE = "Produces from a SAM or BAM a file containing summary alignment metrics." +
```

Comment:
Oh right -- can you point me to an example if the short/long usage syntax?  If you know one off the top of your head, otherwise I'll look for one.